src/connectedk8s/azext_connectedk8s/_client_factory.py

```diff
@@ -19,13 +19,13 @@ def cf_connected_cluster(cli_ctx, _):
     return cf_connectedk8s(cli_ctx).connected_cluster
 
 
-def cf_connectedk8s_prev_2021_04_01(cli_ctx, *_):
-    from azext_connectedk8s.vendored_sdks.preview_2021_04_01 import ConnectedKubernetesClient
+def cf_connectedk8s_prev_2022_05_01(cli_ctx, *_):
+    from azext_connectedk8s.vendored_sdks.preview_2022_05_01 import ConnectedKubernetesClient
     return get_mgmt_service_client(cli_ctx, ConnectedKubernetesClient)
```

Comment:
If you do not need to support multiple API versions, it is recommended that you remove the api-version suffix from the path of the vendored SDK. In this way, you only need to replace the files in the vendored SDK directory in the future, instead of always manually modifying all the SDK paths in the code

Reply:
We need to support multiple api-versions. One is the GA api (2021-10-01) for GA features and another preview api (2022-05-01-preview) for preview features.

Reply:
Got it, thanks